sdk/node/src/AffinityWallet.ts

```diff
@@ -0,0 +1,87 @@
+import { EventComponent } from '@affinidi/affinity-metrics-lib'
+import { CommonNetworkMember as CoreNetwork, __dangerous } from '@affinidi/wallet-core-sdk'
+
+import platformEncryptionTools from './PlatformEncryptionTools'
+
+export type SdkOptions = __dangerous.SdkOptions & {
+  issueSignupCredential?: boolean
+}
+
+const COMPONENT = EventComponent.AffinidiBrowserSDK
```

Comment:
I think this is the enum that is used when sending SDK metrics. Did we miss creating a new enum and using it here?

Reply:
This was intended to be a quick fix to unblock work on BBS task. You're right, this enum should get a new value, and I'm going to do this after I'll return from vacation

Reply:
(The problem here is that EventComponent is defined in affinity-metrics-lib which is another project and package entirely)

Reply:
Please don't forget to take a note / create task for it.